packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -2172,41 +2173,44 @@ curl -v -X DELETE \
 
 `204 No Content`
 
-## Factor Verification Operations
+## Factors that perform Challenge and Verify Operations
```

Comment:
Is it right to say that the factors are the things performing the operations?

Reply:
You're right. I'll fix